advocacy_docs/partner_docs/CommVaultGuide/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+Commvault enables your business to streamline management of its continuously evolving data environment, whether the data is on-premises or in the cloud.
+
+Commvault PostgreSQL iDataAgent provides the flexibility to backup the PostgreSQL/EDB Postgres Advanced Server databases in different modes and restore them in minimal time. You can perform a full/log backup or restore of database servers/individual databases/archive logs at any time and have full control over the process.
```

Comment:
Change to -
to backup PostgreSQL, EDB Postgres Advanced Server and EDB Postgres Extended Server databases in different modes and restore them when needed.